crates/turbopack-ecmascript/src/code_gen.rs

```diff
@@ -26,6 +28,15 @@ pub trait CodeGenerateable {
     fn code_generation(&self, context: EcmascriptChunkingContextVc) -> CodeGenerationVc;
 }
 
+#[turbo_tasks::value_trait]
+pub trait CodeGenerateableWithAvailabilityInfo {
```

Comment:
If there's no harm, could we eliminate this and use a default method impl in `CodeGenerateable` trait?